### PR TITLE
feat(indexers): add Sharewood

### PR DESCRIPTION
**Pull request by Mavyre**
Added the French Sharewood indexer on autobrr. Closes my own Request #966.

### CODE REVIEW

File: internal/indexer/definitions/sharewood.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    required: true
+    label: Passkey
+    help: "Aller sur Mon Profil, puis cliquer sur Afficher Passkey"
```
Comment (s0up4200) [inline internal/indexer/definitions/sharewood.yaml:19]: This should be in English.

```suggestion
    required: true
    label: Passkey
    help: "Go to My Profile, then click on Show Passkey"
```

**Comment by zze0s** [comment]
> By the way, it seems that by default, it uses SASL authentication. From my tests, the authentication doesn't work using SASL but only NickServ. Is there any way to force it in the definition?

There's no way to do that currently, but it's something I've been thinking of adding for some time!

**Review by zze0s — APPROVED** [review]

**Comment by Mavyre** [comment]
> There's no way to do that currently, but it's something I've been thinking of adding for some time!

Can we add a tooltip or a disclosure somewhere to tell the user to change from SASL to NickServ?

**Comment by zze0s** [comment]
> Can we add a tooltip or a disclosure somewhere to tell the user to change from SASL to NickServ?

Could do it in the NickServ Account help text. `Change Mechanism from SASL to NickServ` or if you create a setup guide on the indexers wiki/forum and point it out.

**Comment by Mavyre** [comment]
@zze0s It seems like Sonarr/Radarr both tend to refuse the release pushed by autobrr, because they don't recognise the language. Is it possible, like in prowlarr, to consider MULTi releases as some language? In Prowlarr, I had to setup "Replace MULTi by this language" to MULTi FRENCH to allow Sonarr/Radarr to accept the language upgrade.

Should something like that be configurable in autobrr for "foreign" indexers?

**Comment by zze0s** [comment]
> @zze0s It seems like Sonarr/Radarr both tend to refuse the release pushed by autobrr, because they don't recognise the language. Is it possible, like in prowlarr, to consider MULTi releases as some language? In Prowlarr, I had to setup "Replace MULTi by this language" to MULTi FRENCH to allow Sonarr/Radarr to accept the language upgrade.
> 
> Should something like that be configurable in autobrr for "foreign" indexers?

We currently have nothing in place to be able to do that but there's other indexers that need similar features as being able to hardcode categories and other values, as well as parse titles differently. It's on the todo but no ETA currently.

**Comment by zze0s** [comment]
Related issue #1

**Pull request merged**